### PR TITLE
Upgrade to rollup v2

### PR DESCRIPTION
**Pull request by soupette**
### What does it do?

It migrates the helper-plugin rollup config to rollup v2.

### CODE REVIEW

File: packages/core/helper-plugin/rollup.config.js [hunk 1]
```diff
@@ -1,58 +1,69 @@
...
+      file: packageJson.main,
       format: 'cjs',
-      sourceMap: false,
-      name: '@strapi/helper-plugin',
-      compact: true,
-      globals: {
-        react: 'React',
-        'react-dom': 'ReactDOM',
-      },
+      sourcemap: true,
```
Comment (HichamELBSI) [inline packages/core/helper-plugin/rollup.config.js:18]: I'm not familiar with the rollup config, why do we set the sourcemap value to true now?

File: packages/core/helper-plugin/rollup.config.js [hunk 2]
```diff
@@ -1,58 +1,69 @@
...
-    resolve(),
     babel({
+      presets: [
+        [
+          '@babel/preset-env',
+          {
+            modules: false,
+            loose: true,
+            targets: {
+              browsers: ['Since 2017'],
```
Comment (HichamELBSI) [inline packages/core/helper-plugin/rollup.config.js:50]: Can you explain please that choice? (Sorry for being a rollup newbie 🙈 )

**Review by HichamELBSI — CHANGES_REQUESTED** [review]

**Comment by HichamELBSI** [comment]
Good job @soupette !

File: packages/core/helper-plugin/rollup.config.js (hunk 1 above)
Comment (soupette) [reply packages/core/helper-plugin/rollup.config.js:18]: You areright it should be set to false!

File: packages/core/helper-plugin/rollup.config.js (hunk 2 above)
Comment (soupette) [reply packages/core/helper-plugin/rollup.config.js:50]: I have remove the `.babelrc` so the babel config is set directly using the rollup babel plugin. I might revert that if we create a common package to build all our plugins/lib using webpack. But for the moment I thought it was better to remove it.

**Pull request merged**